haystack/reader/farm.py

```diff
@@ -493,33 +493,34 @@ def eval(
                 "context": doc.text
             }
             # get all questions / answers
-            aggregated_per_question: Dict[str, Any] = defaultdict(list)
+            aggregated_per_question: Dict[tuple, Any] = defaultdict(list)
+            key_tuple = (label.id, label.question)
```

Comment:
Can we rename `key_tuple` to something more meaningful? I would have no clue if I came across this somewhere later in the code! Don't really have a great alternative in mind, but even `question_identifier` or `unique_question_id` could work.

Reply:
Yup renamed to id_question_tuple